src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  function sum(a) {
+    return function(b) {
+      counter++;
+
+      if (counter <= 3 || (counter > 3 && counter % 2 !== 0)) {
+        return a + b;
+      }
```

Comment:
```suggestion
      if (counter > 3 && counter % 2 === 0) {
        return 'Bzzz... Error!';
      }
```

lets make it work like this

Reply:
Hi, Stanislav! You're right, that is shorter) Thanks, done!